src/static/css/main.less

```diff
@@ -4,16 +4,16 @@
    ========================================================================== */
 
 // import Normalize before all other LESS/CSS
-@import "../../vendor/normalize-css/normalize.css";
+@import (less) "../../vendor/normalize-css/normalize.css";
```

Comment:
Sorry, it's too far gone. If I recall correctly, there was some kind of thing where if you didn't import a CSS file that way, that CSS file wouldn't be available for use in the Less compilation. The only example I can think of where this would matter is if you wanted to include a class name from that CSS file as a basic mixin somewhere in your Less. As I think about it, this might have been something we needed when we used Font Awesome. We were importing its compiled CSS (this might've been before it offered its whole set of Less files), but we wanted to use the `.icon-*` classes as mixins somewhere in our Less. And once we started importing it with `(less)` there, we probably just added it to all our imports for safety/consistency.

I'd suggest testing without them and see if they're needed at all anymore.